jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -139,17 +137,15 @@ public void testGreaterOrEqualsVersionWhenFalse() {
         assertThat(result3).isFalse();
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    @Ignore
-// TODO: Remove when implementation is fixed
+    @Test
     public void testGreaterOrEqualsVersionCornerCase() {
         //Given
         String versionA = "asdw4.0.2";
         String versionB = "3.1.1.0{0.1}";
         //When
         String result = EnvUtil.extractLargerVersion(versionA, versionB);
         //Then
-        fail("Exception should have thrown");
+        assertThat(result).isNull();
```

Comment:
You should modify the implementation to throw exception when version is invalid

Reply:
The issue changes the spec to:

> ### Expected Behavior
> 
> `null` should be returned if given versions are invalid.

Maybe we can elaborate on the reasoning behind this

Reply:
I just updated the Issue description to be clearer:
![image](https://user-images.githubusercontent.com/488391/178217992-af0c4416-5ce0-40bf-a5f7-469c9783439d.png)